cmd/cmd.go

```diff
@@ -321,6 +321,14 @@ func Run(Retry bool, showStats bool, cmd *cobra.Command, f func() error) {
 		}
 	}
 
+	// add hook to wind down backends
```

Comment:
I think this should go here

https://github.com/rclone/rclone/blob/326c43ab3f5c7c8570bf87b4656b41846ce00877/fs/cache/cache.go#L22-L27